packages/vue3-flicking/demo/App.vue

```diff
@@ -71,16 +71,12 @@
           <span class="flicking-arrow-next"></span>
         </template>
       </Flicking>
-      <Flicking :plugins="arrow">
+      <Flicking :plugins="fade" :options="{ renderOnlyVisible: true }">
```

Comment:
Can't the demo use both fade and arrow?

Reply:
It can, but I just separated it to test the combination of fade + renderOnlyVisible